ci-runner/executor/stage/ciStages.go

```diff
@@ -325,6 +326,9 @@ func (impl *CiStage) runCIStages(ciContext cicxt.CiContext, ciCdRequest *helper.
 	}
 	return artifactUploaded, nil
 }
+func setDigestInGlobalEnv(scriptEnvs *util2.ScriptEnvVariables, digest string) {
+	scriptEnvs.SystemEnv["DIGEST"] = digest
+}
```

Comment:
we can this method common by passing on the key in input, also after it can be moved to helper